app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+import shutil
```

Comment:
Don't use extra libraries. There is no profit here of using it, you can do exactly the same using os module